packages/cozy-doctypes/src/File.js

```diff
@@ -23,6 +23,51 @@ class CozyFile extends Document {
     const parentDirectoryPath = trimEnd(parentDir.path, '/')
     return `${parentDirectoryPath}/${name}`
   }
+
+  /**
+   * Move file to destination.
+   *
+   * @param   {string} fileId          - The file's id (required)
+   * @param   {string} destinationId   - The target directory's id (required)
+   * @param   {string} force           - Whether we should overwrite the destination in case of conflict
+   * @param   {string} destinationPath - The target path for the file
+   * @returns {Promise}                - A promise that returns the move action response and the deleted file id (if any) if resolved or an Error if rejected
+   *
+   */
+  static async move(fileId, destinationId, force = false, destinationPath) {
```

Comment:
C'est un peu particulier comme sémantique du move, c'est plutôt moveToDirectory ici.

Qu'est ce qui se passe si le destinationPath ne correspond pas du tout au destinationId ?J'ai l'impression que destinationId et destinationPath soient redondants.

Reply:
C'est probalement une mauvaise idée d'appeler ça `destinationPath`, c'est le path "cible" qui permet de récupérer le fichier en conflit pour l'effacer si on veut forcer le move. Alors que le `destinationId` c'est l'id du dossier de destination. J'ai changé plusieurs fois les noms des paramètres et je n'ai en effet pas gardé la meilleure version ;-)

Reply:
destinationName ?

Reply:
C'est un peu bizarre d'avoir le force avant le destinationPath.

```
static async move(fileId, destination, force=false)

destination.id
destination.name
```

?

Reply:
it's much better now, thanks